src/main/java/com/neuvector/Scanner.java

```diff
@@ -329,6 +339,73 @@ private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
+    static String[] getDockerGroupCmdArgs(String scanReportPath, Boolean scanWithNotRootUser) {
+        String[] cmdGroupArgs = {"", ""};
+        if(!scanWithNotRootUser)
```

Comment:
To clarify.. We are not scanning with root user or not user.. We are either passing the user id to the container or not.. So a better variable name for this can be something like `passUserIdToContainer` or `includeUserId` or `runWithCurrentUser`.. Something similar to this.. `NotRootUser` is not very descriptive.

Reply:
b07523fb0c7fcabffbcfc89aa0fcaabe8a391848